Automattic/pocket-casts-ios
# Fix scroll indicator behind player for long episode lists

Fixes #1072 

When scrolling long episode lists, the vertical scroll bar indicator no longer disappears behind the mini player when you get to the bottom of the list

## To test
1. Open current build and find long episode list (100+ should do)
2. As you scroll to the bottom, notice that the vertical scroll indicator moves beyond the bottom of the episode list. If you scroll down far enough, it will disappear completely.

![IMG_5171](https://github.com/Automattic/pocket-casts-ios/assets/25402814/0fdf206d-dfe5-462c-83e8-7a907bafb980)

3. Build with the changes in this PR and repeat the steps above. Notice that the scroll indicator now stays above the mini player

![image](https://github.com/Automattic/pocket-casts-ios/assets/25402814/e5d03794-c952-4c42-be1e-733421d39689)

4. Repeat the same test with episode list in multi-select mode enabled

## Checklist
